Features/Context/SubContext/Authentication.php

```diff
@@ -53,7 +53,7 @@ public function goToPlatformUiAndLogIn($username, $password)
      * @Given I am logged in as admin on PlatformUI
      * @Given I am logged in as an Editor in PlatformUI
```

Comment:
well, the two sentences does not really fit with with the name of the function now, exactly what did you have in mind here @joaoinacio? Always difficult to get right, but be specific, add context to proposals for changes ;)